src/back/auth/src/repo/repository.cpp

```diff
@@ -0,0 +1,43 @@
+#include "repository.hpp"
+
+#include "table_state.hpp"
+#include "table_session.hpp"
+
+#include "userver/components/component_config.hpp"
+#include "userver/components/component_context.hpp"
+#include <userver/yaml_config/merge_schemas.hpp>
+#include <userver/storages/postgres/component.hpp>
+
+namespace svetit::auth {
+
+/*static*/ yaml_config::Schema Repository::GetStaticConfigSchema()
+{
+	return yaml_config::MergeSchemas<components::LoggableComponentBase>(R"(
+type: object
+description: Main repository component
+additionalProperties: false
+properties:
+  some-var:
+    type: string
+    description: some desc
+)");
+}
+
+Repository::Repository(
+		const components::ComponentConfig& conf,
+		const components::ComponentContext& ctx)
+	: components::LoggableComponentBase{conf, ctx}
+	, _pg{ctx.FindComponent<components::Postgres>("database").GetCluster()},
+	_state{svetit::auth::table::State(_pg)},
+	_session{svetit::auth::table::Session(_pg)}
```

Comment:
```suggestion
	, _pg{ctx.FindComponent<components::Postgres>("database").GetCluster()}
	, _state{_pg}
	, _session{_pg}
```

Предлагаю запятые ставить в начале это позволяет менять меньше строк при добавлении новых членов класса и проще менять порядок если он вдруг поменяется в определении.

Reply:
поменял везде.